src/main/kotlin/app/revanced/patches/youtube/layout/branding/icon/CustomBrandingIconPatch.kt

```diff
@@ -15,7 +15,7 @@ import java.nio.file.Files
 @Suppress("DEPRECATION", "unused")
 object CustomBrandingIconPatch : BaseResourcePatch(
     name = "Custom branding icon YouTube",
-    description = "Change the YouTube launcher icon to the icon specified in options.json.",
+    description = "Changes the YouTube app icon to the icon specified in options.json.",
```

Comment:
"app" is not necessarily more appropriate or correct. I made this change because YTM uses this wording. LMK if you prefer "launcher" for both patch descriptions.